spec/features/admin_loads_councillors_spec.rb

```diff
@@ -59,7 +59,7 @@
 
   context "when some councillors have been removed from office" do
     around do |test|
-      Timecop.freeze(2016, 10, 11) { test.run }
+      Timecop.freeze(2016, 10, 15) { test.run }
```

Comment:
This doesn't seem relevant to these changes? I think we added this as a test while we were pairing, but @henare has fixed the timezone issue with #1171 . We should remove this change from the branch.